components/scripts/build-tokens.js

```diff
@@ -10,7 +10,15 @@ function transformTokens(parentKey, object) {
   return objectKeys.reduce((tokensTransformed, objectKey) => {
     const value = object[objectKey]
 
-    if (typeof value === 'object') {
+    if (Array.isArray(value)) {
+      const customProperty = parentKey
+        ? `${parentKey}-${objectKey}`
+        : `${objectKey}`
+
+      return `${tokensTransformed}\n\t--${toKebabCase(
+        customProperty
+      )}: ${value.join()};`
```

Comment:
```suggestion
      )}: ${value.join(', ')};`
```
@matias4205 Podrías verificar si así, los pone separados con `,` y espacio, y si una fuente tipografica tiene espacios, tenemos que añadirle comillas. Pero eso lo podemos resolver el martes creando una función que lo verifique.

Reply:
Así es, ahora lo genera con espacios.